lib/tapioca/cli.rb

```diff
@@ -22,7 +22,7 @@ class Cli < Thor
     class_option :generate_command,
                   aliases: ["--cmd", "-c"],
                   banner: "command",
-                  desc: "The command to run to regenerate RBI files"
+                  desc: "[DEPRECATED] The command to run to regenerate RBI files"
```

Comment:
This will be visible if the user runs `tapioca help` or similar. For proper deprecation, it would be great if all the methods that accept this option could show a warning that the option is deprecated and will be removed if the option is supplied.